server/data/prisonApi.ts

```diff
@@ -29,15 +29,15 @@ class PrisonApi extends RestClient {
   }
 
   async getUserCaseLoads(): Promise<Array<CaseLoad>> {
-    return this.get({ path: '/api/users/me/caseLoads' }) as Promise<Array<CaseLoad>>
+    return (await this.get({ path: '/api/users/me/caseLoads' })) as Promise<Array<CaseLoad>>
```

Comment:
you don't actually need to do this. because the function is async so is awaited elsewhere, adding await just before a return means there's a double-await on the same promise. this explains it better than i do: https://eslint.org/docs/rules/no-return-await

additionally, adding the await keyword means that the type assertion at the end ought to be `Array<CaseLoad>` because the promise has already been resolved.